src/sentry/integrations/slack/webhooks/action.py

```diff
@@ -337,7 +338,51 @@ def build_resolve_modal_payload(self, callback_id):
             "callback_id": callback_id,
         }
 
-    def build_archive_modal_payload(self, callback_id):
+    def build_resolve_modal_payload(self, callback_id) -> View:
+        formatted_resolve_options = []
+        for text, value in RESOLVE_OPTIONS.items():
+            formatted_resolve_options.append(
+                {
+                    "text": {
+                        "type": "plain_text",
+                        "text": text,
+                        "emoji": True,
+                    },
+                    "value": value,
+                }
+            )
```

Comment:
Consider rewriting with a comprehension:

```suggestion
        formatted_resolve_options = [
            {
                "text": {
                    "type": "plain_text",
                    "text": text,
                    "emoji": True,
                },
                "value": value,
            }
            for text, value in RESOLVE_OPTIONS.items()
        ]
```